src/main/java/core/basesyntax/service/filehandler/WriteToFile.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.filehandler;
+
+import core.basesyntax.db.CurrentData;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteToFile {
+    private final CurrentData currentData;
+
+    public WriteToFile(CurrentData currentData) {
+        this.currentData = currentData;
+    }
+
+    public void write(String filePath) {
```

Comment:
This method should take 2 arguments: String  filePath and String report, it is SRP violation to generate report on FileWriter